Makefile

```diff
@@ -64,9 +64,13 @@ clean:
 .PHONY: test
 # run all tests
 test:
```

Comment:
Could you use this target in the github actions file?

Reply:
I think so! I did a quick Stack overflow search and found [this](https://stackoverflow.com/questions/66918575/how-to-use-your-own-makefile-in-github-actions) Ill make the change now!

Reply:
I think you would only need to change [this](https://github.com/project-kessel/inventory-api/blob/main/.github/workflows/build-test.yml#L27) line to `make test`

Reply:
I changed [this](https://github.com/project-kessel/inventory-api/blob/a9a0d0c6a1df3f869ca653230802b2c02c7e0ea9/.github/workflows/build-test.yml#L29) line and checked CI, I can see the total test coverage [here](https://github.com/project-kessel/inventory-api/actions/runs/10391528735/job/28774641459#step:6:61)